src/main/java/rs/teslaris/core/model/document/PersonDocumentContribution.java

```diff
@@ -19,10 +19,10 @@ public class PersonDocumentContribution extends PersonContribution {
     private DocumentContributionType contributionType;
 
     @Column(name = "main_contributor", nullable = false)
-    private boolean mainContributor;
+    private boolean isMainContributor;
```

Comment:
I was thinking that lombok might add the "is" prefix based on boolean type

Reply:
It does, this was left behind from model creation phase and never taken into consideration.